### PR TITLE
[DOC] Fix, updating documentation by update Pimple #982

### PR DESCRIPTION
**Pull request by petrgrishin**

### CODE REVIEW

File: doc/services.rst [hunk 1]
```diff
@@ -104,8 +104,19 @@ And to retrieve the service, use::
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+By default, each time you get a service, returns the same instance of it.
+If you want a different instance to be returned for all calls, wrap your
+anonymous function with the factory() method::
+
+    $app['some_service'] = $app->factory(function () {
+        return new Service();
+    });
+
+Now, each call to ``$app['some_service']`` returns a new instance of the session.
```
Comment (mTorres) [inline doc/services.rst]: returns a new instance of the service.

**Comment by fabpot** [comment]
Thank you @petrgrishin.

**Comment by DsXack** [comment]
:+1:

File: doc/services.rst (hunk 1 above)
Comment (petrgrishin) [inline doc/services.rst]: Thank you @mTorres

**Pull request merged**